### ISSUE TITLE
General

### ISSUE THREAD

**Issue by belmahadzimusic** [seeba-se/Prep-Haven#44]
When product is added to cart and "Continue shopping" is selected from popup, it brings user to homepage rather than keeping them on the same product page or at least bringing them to all products page

**Comment by MelikaH-seeba** [comment]
_When clicked on cart, it leads to cart page, not showing a dropdown like on design.
 There are no filters on products page like on design._
 
 we made a deal for these two to develop it like this, with no dropdown for the first one and for the second one we have only categories since these products are pretty simple, no additional attributes for them to be filtered by, so this is the end result.
 
Regarding last one, where is this the case? To me it changes to the - sign.
<img width="1203" alt="Screenshot 2021-06-16 at 15 07 26" src="https://user-images.githubusercontent.com/57252386/122225001-f385cf00-ceb4-11eb-8c51-5c84c21e995d.png">

@belmahadzimusic

**Comment by belmahadzimusic** [comment]
Checked again and it changes now, however when it's clicked, it does not close the card @MelikaH-seeba

**Comment by MelikaH-seeba** [comment]
yes, it will only close if something else is opened, in that way we assured that the user "gets the context" (when there is a minus, the card is opened and the answer is shown)...and page doesn't look dull. @belmahadzimusic
